### ISSUE TITLE
nuxt babel-runtime-error

### ISSUE THREAD

**Issue by rollakal** [Zulko/eagle.js#81]
Getting this error when tyring to use nuxt:

This dependency was not found:                                                                        
* babel-runtime/core-js/object/assign in ./node_modules/eagle.js/dist/eagle.es.js

sombody had a similar isssue and knows how to fix it?

**Comment by yaodingyd** [comment]
I'm not familiar with nuxt but I think you should use the commonjs distribution of eagle.js when builidng with nuxt.

**Comment by yaodingyd** [comment]
close for inactivity

**Comment by bgondy** [comment]
Same issue here with a clean VueJS project created with Vue CLI (without nuxt):

I've installed `eagle.js` with npm.

```js
import Vue from 'vue';
import App from './App.vue';
import Eagle from 'eagle.js';
import 'eagle.js/dist/eagle.css';

Vue.config.productionTip = false;

Vue.use(Eagle);

new Vue({
  render: h => h(App),
}).$mount('#app');
```

When i run `npm run serve`:

```
 ERROR  Failed to compile with 1 errors                                                                             12:28:07 AM

This dependency was not found:

* babel-runtime/core-js/object/assign in ./node_modules/eagle.js/dist/eagle.es.js

To install it, you can run: npm install --save babel-runtime/core-js/object/assign 
```

If I try to install it, here is what I get:

```
➜  eaglejs-project git:(master) ✗ npm install --save babel-runtime/core-js/object/assign
npm ERR! code ENOLOCAL
npm ERR! Could not install from "babel-runtime/core-js/object/assign" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bgondy/.npm/_logs/2019-05-25T22_30_07_866Z-debug.log
```

I think it is due to breaking changes introduced with Babel 7.0.0 and package renames (@babel/runtime). WDYT ?

If I remove the import statement in `dist/eagle.es.js`, I don't get any error (but also no polyfill):
```js
import _Object$assign from 'babel-runtime/core-js/object/assign';
```

**Comment by yaodingyd** [comment]
Interesting, I’ll take a look.

**Comment by yaodingyd** [comment]
still working on update all tooling the dependencies...

**Comment by bgondy** [comment]
Thank you for the update. Installing `babel-runtime` works for now.

**Comment by yaodingyd** [comment]
fixed in 0.6.0
